### DISCUSSION TITLE
If the iperf3 - O parameter exceeds 60 seconds, an error will be reported

### DISCUSSION THREAD

**Discussion by Wuzuqiang666** [esnet/iperf#1302] [Q&A]
# Context

* Version of iperf3: 3.11

* Hardware:Window x86-64
![image](https://user-images.githubusercontent.com/45865497/159439547-7215c020-d122-434a-a49b-d4ac3113372d.png)

**Comment by swlars** [comment]
Hi, Iperf3 is working as intended. The value of --omit should be between 0 and 60.
